input/otlp/logs_test.go

```diff
@@ -532,6 +532,10 @@ func processLogEvents(t *testing.T, logs plog.Logs) modelpb.Batch {
 }
 
 func TestConsumerConsumeLogsDataStream(t *testing.T) {
+	randomString, err := generateRandomString(otlp.MaxDataStreamBytes)
```

Comment:
`generateRandomString` can be replaced by strings.Repeat("abcdefghijklmnopqrstuvwxyz0123456789", 10) then we can get rid of the test util func, remove crypto/rand import, as well as error handling from generating a random string